components/merkledb/examples/blockchain.rs

```diff
@@ -84,7 +84,7 @@ struct Schema<T: Access> {
     pub transactions: MapIndex<T::Base, Hash, Transaction>,
     pub blocks: ListIndex<T::Base, Hash>,
     pub wallets: ProofMapIndex<T::Base, PublicKey, Wallet>,
-    access: T,
+    pub wallet_history: Group<T, PublicKey, ProofListIndex<T::Base, Hash>>,
```

Comment:
@popzxc Does this form for index groups looks better? In the same vein, it could be possible to implement `Lazy<T, Index<T, ..>>` wrapper, but I'm not sure whether it makes sense.

Reply:
Hm... Yeah, I think it's better, but nevertheless the overall verbosity of the construction (double template, `T`, `T::Base`) looks scary. On the other hand, this can be overrided with type aliases I guess.